### ISSUE TITLE
How can you be productive with git-secret - how to manage gitignore?

### ISSUE THREAD

**Issue by ieugen** [sobolevn/git-secret#219]
Hi, 

I'm finding it very dificul to use this program with managing .gitignore and not ignoring *.secret files? 
I have a deep structure and in the leaf directory lots of files if I ignore all files but not ignore .secret files. 

How do you do it sanely? I did not expect to be required to add .secret files to git and when I cloned the repo it failed. Why not move the secret files under .gitsecret or managet .gitignore by default !?! 

Thanks, 
````
.
├── production
│   ├── db-pass
│   ├── db-pass.secret
│   ├── keystore.jceks
│   ├── keystore.jceks.secret
│   ├── mailgun-secret-key
│   ├── mailgun-secret-key.secret
│   ├── nexus-password
│   ├── nexus-password.secret
│   ├── nexus-user
│   ├── nexus-user.secret
│   ├── settings.xml
│   └── settings.xml.secret
└── staging
    ├── db-pass
    ├── db-pass.secret
    ├── keystore.jceks
    ├── keystore.jceks.secret
    ├── mailgun-secret-key
    ├── mailgun-secret-key.secret
    ├── nexus-password
    ├── nexus-password.secret
    ├── nexus-user
    ├── nexus-user.secret
    ├── settings.xml
    └── settings.xml.secret
````

.gitignore
`````
??????? 
`````

**Comment by ieugen** [comment]
I figured it out. It seems git ignore and globs need to be explicit

.gitignore
````
charts/api-secrets/config/staging/*
charts/api-secrets/config/production/*
!charts/api-secrets/config/staging/*.secret
!charts/api-secrets/config/production/*.secret
````

**Comment by joshrabinowitz** [comment]
@ieugen - this seems like a .gitignore issue, not a git-secret issue, and you've gotten it worked out. .OK if I close this issue?

**Comment by eduncan911** [comment]
Came here to file an issue and saw this, and i have to say yes it is very disrupted to step out of `git secret add` to go edit a `.gitignore`.

Opened issue #220 about it should be auto-added per the docs

**Comment by ieugen** [comment]
Hi @eduncan911 , I did not know the behavior was to add files to .gitigore. 
@joshrabinowitz : From my point of view, this issue should be closed as #220 is the main cause.
